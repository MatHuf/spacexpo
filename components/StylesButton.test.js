import React from "react";
import renderer from "react-test-renderer";

import Button from "./StyledButton";

const props = {
  title: "Press Here",
  onPress: () => {}
};

describe("Button", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(React.createElement(() => Button(props)))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
