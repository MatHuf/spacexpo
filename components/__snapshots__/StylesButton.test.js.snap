// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#000000",
      "color": "#EEEEEE",
      "opacity": 1,
      "padding": 10,
    }
  }
>
  <Text>
    \`$
    Press Here
    \`
  </Text>
</View>
`;
